### PR TITLE
reduce font size and rm unnessary gap - email popup

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**

### CODE REVIEW

**Review by Linhhnk — APPROVED** [review]

File: src/components/SubscribeButton/NotificationModal.tsx [hunk 1]
```diff
@@ -184,7 +184,7 @@ export default function NotificationModal({
         )
       case VIEW.UNSUBSCRIBE:
         return (
-          <>
+          <div>
```
Comment (viet-nv) [inline src/components/SubscribeButton/NotificationModal.tsx:187]: why do you add a redundant div here?

File: src/components/SubscribeButton/NotificationModal.tsx [hunk 2]
```diff
@@ -198,11 +198,11 @@ export default function NotificationModal({
                 </ButtonTextt>
               </ButtonOutlined>
             </ActionWrapper>
-          </>
+          </div>
         )
       case VIEW.SUBSCRIBE_SUCCESS:
         return (
-          <>
+          <div>
```
Comment (viet-nv) [inline src/components/SubscribeButton/NotificationModal.tsx:205]: why do you add a redundant div here? what happen with fragment

Comment (nguyenhoaidanh) [reply src/components/SubscribeButton/NotificationModal.tsx:205]: I don't need gap effect by it's parent.

Comment (viet-nv) [reply src/components/SubscribeButton/NotificationModal.tsx:205]: then, just remove gap from `ContentWrapper` https://github.com/KyberNetwork/kyberswap-interface/pull/1477/files#diff-bb86fa0ce5436db6d998b2222602c8b4ea74c570b8d2bd3bca2bf91d4c5bf3acR28

Comment (nguyenhoaidanh) [reply src/components/SubscribeButton/NotificationModal.tsx:205]: <Flex gap=...>
.... 
I need gap here
<div>
   my component here, I don't need gap here
</div>
</Flex>

Comment (viet-nv) [reply src/components/SubscribeButton/NotificationModal.tsx:205]: okay, got it

**Review by viet-nv — APPROVED** [review]

File: src/components/SubscribeButton/NotificationModal.tsx (hunk 2 above)
Comment (nguyenhoaidanh) [reply src/components/SubscribeButton/NotificationModal.tsx:205]: thank you

**Pull request merged**